### ISSUE TITLE
R is in red in the entire code

### ISSUE THREAD

**Issue by vasoolibhai** [Dheeraj22/Professor_DK#4]
At, the start I connected my app with firebase by adding codes in grandle and adding json file in projects-> app. Now, the problem is that whenever I make a new empty R goes on red before adding any activity everything is fone what is going wrong plz help. What i think is after attaching your app to firebase and adding json file and copying grandle codes we cannot add any new activity to do so u need to add all the activities before it am i right?

**Comment by punjabweb** [comment]
Yes same with me.. I have started learning Android Studio 3 and this code shows many errors. Build Failed etc. It does not show up on emulator either.

**Comment by zyllus17** [comment]
Delete the R and write it again, a list will open. Then press enter,it will be fixed or simply place the pointer at red R and press ALt+Enter. It will be fixed
